test/functional/fixtures/regression/gh-850/test.js

```diff
@@ -0,0 +1,7 @@
+describe('[Regression](GH-850)', function () {
+    it('Should resume the test if the cursor is moving from reloaded iframe to top window', function () {
+        // NOTE: we set selectorTimeout to a large value to wait for an iframe to load
+        // on the farm (it is fast locally but can take some time on the farm)
+        return runTests('testcafe-fixtures/index.test.js', 'Move from reloaded iframe', { selectorTimeout: 3000 });
```

Comment:
It's all ok, just increase please `selectorTimeout` for this tests to avoid the problems with slow connection on saucelabs vms (use 10 sec to be sure)